bramkouwen/Homework-1
# Feature - Updating the index copy

## What does this PR do?
* It updates the index.html to contain paragraphs of text under each sub-header

## How do you test the PR?
* Pull down the branch 
* Visit the site in your browser and see the added paragraphs of text

## Screenshots
None
